### DISCUSSION TITLE
v7.1.1

### DISCUSSION THREAD

**Discussion by Taiko2k** [Taiko2k/Tauon#688] [General]
**v7.1.1** *Its better now edition*

Changes:

 - There are now new indicators on tabs when dragging tracks to show if any track is in that playlist already (yellow) or if the playlist is a generator playlist therefore can't add tracks (red).
   <img width="551" alt="Screenshot 2022-02-14 at 11 45 46 AM" src="https://user-images.githubusercontent.com/17271572/153778767-ed6402e5-c972-4f87-9b09-0c600be47117.png">
 - There is now a function prompt under MENU to clean the database. You should click this if you see it to cut down on unnecessary load times.
 - HiDPI scaling setting now applies changes immediately without needing restart.
 - [Wayland] UI will now try automatically scale when moving between monitors (Experimental)
 - [Phazor] Internet radio stations start faster. (Reduced the initial buffer size)
 - [Phazor] Possibly fixed some various rare issues with internet radio.

Feature deprecation:
 - Spotify support. 
 The main issue is the search provided by the API being somewhat useless. If you still used this feature then please get in touch and submit feedback. 

:dollar: As always, consider becoming a supporter if aren't already. It goes a long way with helping motivate me to continue working on this app!

<hr /><em>This discussion was created from the release <a href='https://github.com/Taiko2k/TauonMusicBox/releases/tag/'>v7.1.1</a>.</em>

**Comment by Solarunit** [comment]
Nice update!
Does indicators on tabs work the same in the playlist side panel?

**Reply by Taiko2k** [reply]
Yes it's intended to

**Reply by nicolnt** [reply]
Yes it's awesome for making playlists, so convenient now!
